### PR TITLE
fixed argument parser

### PR DESCRIPTION
**Pull request by land-pack**
If user add params at the end of url with `/demo/api?name=frank`. When we try to parser the url by use `GetString`. and we got `string=frank?name=frank`. so I guess we need to just parser it to `frank`.

### CODE REVIEW

File: controller.go [hunk 1]
```diff
@@ -296,7 +296,7 @@ func (c *Controller) Redirect(url string, code int) {
 	c.Ctx.Redirect(code, url)
 }
 
-// SetData set the data depending on the accepted
+// Set the data depending on the accepted
```
Comment (astaxie) [inline controller.go]: please fix this

File: controller.go [hunk 2]
```diff
@@ -370,7 +370,7 @@ func (c *Controller) ServeXML() {
 	c.Ctx.Output.XML(c.Data["xml"], hasIndent)
 }
 
-// ServeYAML sends yaml response.
+// ServeXML sends xml response.
```
Comment (astaxie) [inline controller.go]: please fix this

**Comment by astaxie** [comment]
How do you use the `GetString`?

**Comment by land-pack** [comment]
> How do you use the `GetString`?

Like What I said in my commit message.

File: controller.go (hunk 2 above)
Comment (astaxie) [reply controller.go]: ?

**Comment by astaxie** [comment]
test failed

**Comment by flycash** [comment]
it was suspended for a long time. I closed it.

**Pull request closed without merging**